driver/modern_bpf/definitions/events_dimensions.h

```diff
@@ -160,6 +160,8 @@
 #define IO_URING_SETUP_X_SIZE HEADER_LEN + sizeof(int64_t) + sizeof(uint32_t) * 7 + PARAM_LEN * 8
 #define MMAP2_E_SIZE HEADER_LEN + sizeof(uint64_t) * 3 + sizeof(int64_t) + sizeof(uint32_t) * 2 + PARAM_LEN * 6
 #define MMAP2_X_SIZE HEADER_LEN + sizeof(int64_t) + sizeof(uint32_t) * 3 + PARAM_LEN * 4
+#define SEMGET_E_SIZE HEADER_LEN + sizeof(int32_t) * 2 + sizeof(uint32_t) + PARAM_LEN * 3
+#define SEMGET_X_SIZE HEADER_LEN + sizeof(int64_t) + PARAM_LEN 
```

Comment:
Just wanted to notice this: I would put the PARAM_LEN before each sizeof, so to better reflect the layout of the event in memory (header, lengths, actual params). Just a nit-pick, nothing important

Reply:
agree with that, one day we could also move these sizes to the event table or even better generate them at compile time

Reply:
Yep exactly, I was thinking about how to generate them too :)